controllers/router_controller.py

```diff
@@ -0,0 +1,30 @@
+import http
+import json
+
+from fastapi import APIRouter, Depends
+from fastapi.responses import JSONResponse
+from controllers import passenger
+from model.requests.passenger import PassengerDriveOrderRequest, PassengerGetDrive
+
+router = APIRouter()
+
+# TODO handle backend routing
+
+@router.post("/newDriveOrder")
```

Comment:
make sure it is similar to what I did in other endpoints. if it is, than it's fine

Reply:
This routing has nothing to do with your endpoints, its between FE and BE :) I'm using your routings in the driver's mode (differrant PR)

Reply:
I meant that the route text "newDriveOrder" is not necessarily the letter-case we use in this component.
For example, when Aviv created users_map controller, the token validation endpoint is something like: /users/validate_token and not users/validateToken.
We need to stick to one case. I don't care which as long as it is unified.